src/librustc_mir/interpret/operand.rs

```diff
@@ -242,13 +248,11 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
             }
         };
 
+        let alloc = self.memory.get_raw(ptr.alloc_id)?;
```

Comment:
This is just a no-behavioral-change optimization, right?

Reply:
Ah yes, initially I thought I could merge it with the above `get_raw` call, I'll pull it out into a separate commit

Reply:
Oh :laughing: it is already a separate commit